CHANGELOG.md

```diff
@@ -6,25 +6,44 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.1.0] - 2021-04-07
 ### Added
-- Simple authorization through a login page
-- Ability to upload a PDF to the server
-- Ability to download and view a PDF from the server
-- Ability to receive push notifications
-- Runnable on Android and Web
+- Ability to manage users

```

Comment:
This section is for things that are allready implemented but not yet released. We have not yet implemented these features at all.